games-misc/parsec-bin/parsec-bin-147.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit gnome2-utils unpacker
```

Comment:
Sorry its taken forever. You should be able to update EAPI to 7 if you drop gnome2-utils and use xdg-utils instead. You can then call `xdg_icon_cache_update` below. Please update and test if you still wish to maintain this. It can now be merged if it still works. 

Also please rebase against fresh master.